recipes/serd/all/conandata.yml

```diff
@@ -6,3 +6,6 @@ sources:
     autowaf:
       url: "https://gitlab.com/drobilla/autowaf/-/archive/aeef9f5fdf416d9b68c61c75de7dae409f1ac6a4/autowaf-aeef9f5fdf416d9b68c61c75de7dae409f1ac6a4.tar.gz"
       sha256: "824f719bb77b9e0ae12fd80fe5b97194add6cd8b877c9fd8723e89d7c1940484"
+  "0.30.14":
```

Comment:
New version was dropped to fix a bug https://gitlab.com/drobilla/serd/-/blob/master/NEWS#L3

Reply:
The versions with uneven patch versions are "rolling" versions. Only versions with a even patch version are immutable. So I would wait for 0.30.16 and create a new PR then. 

Until then I would still like to publish this version as it fixes a quasi-memory leak.

Reply:
Thanks for that info... "mutable releases" are a curse so 👍 on your idea